port_ocean/clients/port/mixins/integrations.py

```diff
@@ -65,12 +100,21 @@ async def create_integration(
             "changelogDestination": changelog_destination,
             "config": {},
         }
-        if port_app_config:
+
+        if use_provisioned_defaults:
+            json["provisionEnabled"] = use_provisioned_defaults
+
```

Comment:
this shouldn't be something on top of the integration, but rather as a query param
see the image below for reference
<img width="737" alt="Screenshot 2025-01-07 at 13 27 02" src="https://github.com/user-attachments/assets/0db830f6-1d55-4eb4-b540-832012e0d6bc" />

Reply:
Fixed